bin.src/rucio-integration-register

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+
+import logging
+import os
+import sys
+
+from lsst.daf.butler import Butler
+from lsst.dm.rucio.register.app_parser import AppParser
+from lsst.dm.rucio.register.rucio_interface import RucioInterface
+from lsst.dm.rucio.register.rucio_register_config import RucioRegisterConfig
+
+logger = logging.getLogger(__name__)
+
+RUCIO_REGISTER_CONFIG = "RUCIO_REGISTER_CONFIG"
+_MSG = "environment variable not set, and no configuration was specified on the command line"
+
+
+def main():
```

Comment:
Policy is to have very short script implementation scripts here so that you have the option of switching from scons to pip for script creation. Move `main` into a `python/lsst/dm/rucio/register/script/_register.py` and you can put the app_parser in there as well. Although I may have misunderstood the hierarchy here if `register` by definition only has one command-line in it then `python/lsst/dm/rucio/register/script.py` would be fine.

Reply:
Fixed